src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Apprentice/Details.cshtml

```diff
@@ -123,6 +123,14 @@
                         <td id="apprenticeship-pause-date" class="govuk-table__cell">@Model.PauseDate.Value.ToGdsFormat()</td>
                     </tr>
                 }
+
+                @if (Model.Status == ApprenticeshipStatus.Completed)
+                {
+                    <tr class="govuk-table__row">
+                        <th scope="row" class="govuk-table__header">Completion payment month</th>
+                        <td id="apprenticeship-completed-date" class="govuk-table__cell">@Model.CompletionDate.Value.ToGdsFormatWithoutDay()</td>
```

Comment:
The screenshot in the ticket seems to indicate that the long month name is required.